examples/sv3d/configs/sampling/sv3d_u.yaml

```diff
@@ -0,0 +1,124 @@
+model:
+  target: models.diffusion.VideoDiffusionEngine
+  params:
+    scale_factor: 0.18215
+    disable_first_stage_amp: True
+#    ckpt_path: ckpt/sv3d_u.ckpt  # do not pass in the ckpt path here, as the ckpt already been init under the main func, passing in here will reinit the model under the diffusion module class
+
+    denoiser_config:
+      target: sgm.modules.diffusionmodules.denoiser.Denoiser
+      params:
+        weighting_config:
+          target: sgm.modules.diffusionmodules.denoiser_weighting.EDMWeighting
+          params:
+            sigma_data: 1.0
+        scaling_config:
+          target: sgm.modules.diffusionmodules.denoiser_scaling.VScalingWithEDMcNoise
+
+    network_config:
+      target: modules.video_model.VideoUNet
```

Comment:
same here. video_*

Reply:
I am neutral about whether to change it, as I tried to follow the official naming here: https://github.com/Stability-AI/generative-models/blob/main/configs/inference/sv3d_u.yaml#L14.